resources/recruiting-board/board-responsibilities.md

```diff
@@ -0,0 +1,60 @@
+
+**Board members are expected to allocate at least 4 hours per week to Foundation tasks.**
+
+## Committee roles and responsibilities
+
+### [Marketing Committee](https://github.com/dotnet-foundation/wg-marketing)
+
+The marketing committee is here to help spread the message of openness in the .NET ecosystem. We support community and project initiatives, drive awareness and increase membership. We create consistent, powerful storytelling to increase share of conversation and establish industry relevance for the .NET Foundation.
+
+The marketing committee is composed of volunteers that are primarily marketers, not developers (although there are some more technical than others). They are developer marketing and technical marketers that care about and/or have bet their businesses on .NET. This is a smaller committee because most people that show interest in volunteering are not interested in marketing and are referred to the outreach committee which is focused on community activities and education.
+
+The main goals of the marketing committee are:
+
+- Increase membership
+- Generate awareness (of .NET open source and the .NET Foundation itself)
+- Foster and grow an open, healthy.NET community
+
+#### Marketing Committee Chair
+
+The marketing committee chair is responsible for directing the committee’s marketing strategy and execution.
+
+**Always-on activities** The chair runs monthly meetings with members and is on point for all announcements and air cover (for dealing with bad PR if needed as it arises). They are responsible for the foundation’s web presence, running all the social handles(Twitter, Facebook, LinkedIn) and growing followers, pulling together and sending the bi-monthly newsletter(MailChimp), driving leads(for newsletter and membership), running the [SWAG store](https://store.dotnetfoundation.org), writing and sending press releases.
+
+**Other activities** They are also responsible for new project “go-to-market” GTM, co-marketing initiatives, event sponsorships & presence, some event management (i.e. .NET Conf), website design & copy,branding and logos, surveys, presentation decks, and coordination with other committees around messaging and awareness as needed.
+
+***Time commitment: 40-60 hrs./month***
+
+## [Outreach Committee](https://github.com/dotnet-foundation/wg-outreach)
+
+The Outreach Committee is here to welcome developers of all backgrounds,education, and technology ecosystems into the .NET ecosystem. We are here to help you organize and evangelize.
+
+High-level goals:
+
+- Encourage new developers to build with .NET
+- Empower underrepresented segments of the coder community to become leaders and contributors
+- Assist event organizers with evangelism and growth
+
```

Comment:
Also can anyone be a part of it as well? Is it paid or unpaid? Are there specific times where we would have to meet if I were to be a part of it? All questions I have for this, I have college so I would need to know if it would conflict with my schedule.

Also do I have to be a member first (currently I am a member but in external CI access so I feel like that is not full membership, just partial)?

Is part of the outreach committee's goals also to "ensure that features that a lot of people would like are part of .NET" so that they would have more motivation to move to .NET as well?

Reply:
Hi @AraHaan 

These are all good questions. I'll create a new issue to address them. (We'll be updating all the committee info in the next month or so).


> Also can anyone be a part of it as well?
Yes.
> Is it paid or unpaid? 
Unpaid. All the work done by the foundation is unpaid.

> Are there specific times where we would have to meet if I were to be a part of it? 
Each committee sets its meeting schedule, taking into account timezones and availability of the committee members.

> What about traveling? 

Not necessary. All the committees meet over teams.

> Also do I have to be a member first (currently I am a member but in external CI access so I feel like that is not full membership, just partial)?

That's full membership. There's a new membership portal that will be launching soon. That may be what you've heard about.

> Is part of the outreach committee's goals also to "ensure that features that a lot of people would like are part of .NET" so that they would have more motivation to move to .NET as well?

No. The .NET Foundation doesn't set the roadmap for any member projects. That's all done by each projects' maintainers.